### PR TITLE
feat: add i18n dictionary 

### PR DESCRIPTION
**Pull request by haha861924**
# CHANGES

- [x] i18n dictionary
- [x]  set i18n

### CODE REVIEW

File: src/locales/zhTW.json [hunk 1]
```diff
@@ -1,9 +1,50 @@
...
+    "gender": "性別",
+    "birth": "生日",
+    "height": "身高(公分)",
+    "weight": "體重(公斤)",
+    "bodyFat": "體脂(%)",
+    "waist": "腰圍(公分)",
+    "activity": "活動量",
+    "pregnancy": "懷孕哺乳狀態",
+    "dailyHeatSuggest" : "每日熱量攝取建議",
+    "waistAndHight": "腰身高比",
```
Comment (HuskyHsu) [inline src/locales/zhTW.json:15]:
```suggestion
    "waistAndHight": "腰圍身高比",
```

File: src/locales/zhTW.json [hunk 2]
```diff
@@ -1,9 +1,50 @@
...
+    "basalMetabolism": "基礎代謝率",
```
Comment (HuskyHsu) [inline src/locales/zhTW.json:16]:
```suggestion
    "basalMetabolism": "基礎代謝率(仟卡)",
```

File: src/locales/zhTW.json [hunk 3]
```diff
@@ -1,9 +1,50 @@
...
+    "bodySurfaceArea":"體表面積",
+    "tdee": "TDEE"
```
Comment (HuskyHsu) [inline src/locales/zhTW.json:18]:
```suggestion
    "tdee": "TDEE(仟卡)"
```

File: src/locales/zhTW.json [hunk 4]
```diff
@@ -1,9 +1,50 @@
...
+    "calculate": "計算",
+    "gender": "性別",
+    "birth": "生日",
+    "height": "身高(公分)",
+    "weight": "體重(公斤)",
+    "bodyFat": "體脂(%)",
+    "waist": "腰圍(公分)",
+    "activity": "活動量",
+    "pregnancy": "懷孕哺乳狀態",
+    "dailyHeatSuggest" : "每日熱量攝取建議",
```
Comment (HuskyHsu) [inline src/locales/zhTW.json:14]:
```suggestion
    "dailyHeatSuggest" : "每日熱量攝取建議(仟卡)",
```

File: src/locales/en.json [hunk 5]
```diff
@@ -1,9 +1,52 @@
...
+    "calculate": "Calculate",
+    "gender": "Gender",
+    "birth": "Birthday",
+    "height": "Height(CM)",
+    "weight": "Weight(KG)",
+    "bodyFat": "Body Fat(%)",
+    "waist": "Waistline(CM)",
+    "activity": "Activity Level",
+    "pregnancy": "Pregnancy Breastfeeding",
+    "dailyHeatSuggest" : "Daily Heat Suggest",
```
Comment (HuskyHsu) [inline src/locales/en.json:14]:
```suggestion
    "dailyHeatSuggest" : "Daily Heat Suggest(kcal)",
```

File: src/locales/en.json [hunk 6]
```diff
@@ -1,9 +1,52 @@
...
+    "waistAndHight": "Waist to Height Ratio",
+    "basalMetabolism": "Basal Metabolic Rate",
+    "bodySurfaceArea":"Body Surface Area",
+    "tdee": "TDEE"
```
Comment (HuskyHsu) [inline src/locales/en.json:18]:
```suggestion
    "tdee": "TDEE(kcal)"
```

File: src/locales/en.json [hunk 7]
```diff
@@ -1,9 +1,52 @@
...
+    "birth": "Birthday",
+    "height": "Height(CM)",
+    "weight": "Weight(KG)",
+    "bodyFat": "Body Fat(%)",
+    "waist": "Waistline(CM)",
+    "activity": "Activity Level",
+    "pregnancy": "Pregnancy Breastfeeding",
+    "dailyHeatSuggest" : "Daily Heat Suggest",
+    "waistAndHight": "Waist to Height Ratio",
+    "basalMetabolism": "Basal Metabolic Rate",
```
Comment (HuskyHsu) [inline src/locales/en.json:16]:
```suggestion
    "basalMetabolism": "Basal Metabolic Rate(kcal)",
```

File: src/locales/en.json [hunk 8]
```diff
@@ -1,9 +1,52 @@
...
+
   },
   "meta": {
     "slogan": "Knowing your health status",
     "source": "Dr. Wang"
+  },
+  "options": {
+    "male": "Male",
+    "female": "Female",
+    "scarcely":"Scarcely",
```
Comment (HuskyHsu) [inline src/locales/en.json:28]: prettier not work or pass json file?

**Review by ashramwen — CHANGES_REQUESTED** [review]

File: src/locales/zhTW.json [hunk 9]
```diff
@@ -1,9 +1,50 @@
...
+    "moderateToSevere":"中重度",
+    "verySevere":"極重度",
+    "firstTrimester":"第一孕期",
+    "secondTrimester":"第二孕期",
+    "thirdTrimester":"第三孕期",
+    "breastfeeding":"哺乳"
+  },
+  "info":{
+    "dailyHeatSuggest": "根據活動量及最低BMR計算",
+    "waistAndHight": "根據活動量及最低BMR計算",
```
[the remark this answers is not in the document]
Comment (haha861924) [reply src/locales/zhTW.json]: image
image

應該是只需要一個

**Review by ashramwen — APPROVED** [review]

**Pull request merged**